manifests/params.pp

```diff
@@ -32,29 +32,55 @@
       }
     }
     'RedHat': {
-      $dnsdir             = '/etc'
-      $vardir             = '/var/named'
-      $optionspath        = '/etc/named/options.conf'
-      $zonefilepath       = "${vardir}/dynamic"
-      $localzonepath      = "${dnsdir}/named.rfc1912.zones"
-      $defaultzonepath    = 'unmanaged'
-      $publicviewpath     = "${dnsdir}/named/zones.conf"
-      $viewconfigpath     = "${dnsdir}/named/views"
-      $dns_server_package = 'bind'
-      $namedservicename   = 'named'
-      $user               = 'named'
-      $group              = 'named'
-      $rndcconfgen        = '/usr/sbin/rndc-confgen'
-      $named_checkconf    = '/usr/sbin/named-checkconf'
-      $sysconfig_file     = '/etc/sysconfig/named'
+      $user = 'named'
+      $group = 'named'
       $sysconfig_template = "dns/sysconfig.${facts['os']['family']}.erb"
       $sysconfig_startup_options = undef
       $sysconfig_disable_zone_checking = undef
 
       # This option is not relevant for RedHat
       $sysconfig_resolvconf_integration = undef
 
-      $dnssec_enable = if versioncmp($facts['os']['release']['major'], '9') >= 0 { undef } else { 'yes' }
+      if $dns::globals::scl {
+        $sclroot = '/opt/isc/isc-bind/root'
+        $sclconfroot = '/etc/opt/isc/scls/isc-bind'
+        $sclenvname = 'isc-bind'
+
+        $dnsdir = $sclconfroot
+        $vardir = '/var/named'
+        $optionspath = "${dnsdir}/options.conf"
+        $zonefilepath = "${vardir}/dynamic"
+        $localzonepath = "${dnsdir}/named.rfc1912.zones"
+        $defaultzonepath = 'unmanaged'
+        $publicviewpath = "${dnsdir}/zones.conf"
+        $viewconfigpath = "${dnsdir}/views"
+        $dns_server_package = 'isc-bind'
```

Comment:
actually I would like to stick to the metapackage as it will install more dependencies which isc-bind-bind will not: 

```
$ yum install isc-bind
...
Installing:
 isc-bind
Installing dependencies:
 checkpolicy
 environment-modules
 isc-bind-bind
 isc-bind-bind-libs
 isc-bind-bind-utils
 isc-bind-fstrm
 isc-bind-fstrm-utils
 isc-bind-libevent
 isc-bind-libuv
 isc-bind-protobuf-c
 isc-bind-runtime
 jemalloc
 policycoreutils-python-utils
 python3-audit
 python3-libsemanage
 python3-policycoreutils
 python3-setools
 scl-utils
 tcl
 
 
--------------------------------------
$ yum install isc-bind-bind
...
Installing:             
 isc-bind-bind          
Installing dependencies:
 environment-modules    
 isc-bind-bind-libs     
 isc-bind-fstrm         
 isc-bind-fstrm-utils   
 isc-bind-libevent      
 isc-bind-libuv         
 isc-bind-protobuf-c    
 isc-bind-runtime       
 jemalloc               
 scl-utils              
 tcl                    
 ```

Reply:
is that OK with u?

Reply:
Perhaps then make it explicit what you're doing (and why). Something like
```suggestion
        # Install the SCL meta package to pull in utils
        $dns_server_package = $sclenvname
```

Reply:
allright - change has been added